lib/ansible/modules/system/hostname.py

```diff
@@ -679,6 +679,12 @@ class DebianHostname(Hostname):
     strategy_class = DebianStrategy
 
 
+class KylinHostname(Hostname):
```

Comment:
Could you please write a test for this change ? You can check existing tests [here](https://github.com/ansible/ansible/blob/devel/test/units/module_utils/test_distribution_version.py).